File: Programming-languages-and-laboratory/Java/Exercises/24_compito_java_2018_07_25/Innovazione.java
import java.io.*;
import java.util.*;

public class Innovazione extends Progetto {

    private int aziende;

    public Innovazione(int cod, String t, String c, String o, int a, double i) {
        super(cod, t, c, o, i);
        aziende = a;
    }

    public String toString() {
        return "innovazione\t" + super.toString() + "-\t-\t" + aziende + "\t" + importo * 1000;
    }
}

File: II - Linguaggi di programmazione e laboratorio/Esame/Python/24_compito_java 20180725/Innovazione.py
from Progetto import Progetto


class Innovazione(Progetto):
    def __init__(self, codice, tipo, titolo, nome, organizzazione, aziende, importo):
        super().__init__(codice, tipo, titolo, nome, organizzazione, importo)
        self._aziende = aziende

    def __str__(self):
        return super().__str__() + "\t-\t-\t" + str(self._aziende) + "\t" + str(self._importo)
